base/applications/games/solitaire/solgame.cpp

```diff
@@ -10,6 +10,7 @@ extern TCHAR MsgWin[128];
 extern TCHAR MsgDeal[128];
 
 CardStack activepile;
+int visible_pile_cards;
```

Comment:
Please keep the current codestyle:
```suggestion
int VisiblePileCards;
```
or something like this...

Reply:
Done!